### PR TITLE
207 custom attributes

### LINKED ISSUE #207: Pass all/custom attributes to filterRequestHeaders
Hi, 

I have noticed that custom  attributes are not passed to `filterRequestHeaders`.
Is possible to pass all of them or there is a specific design decision behind this "restricted" behaviour?

Thanks.

### PR DESCRIPTION
**Pull request by rbarilani**
closes #207

### CODE REVIEW

**Comment by rbarilani** [comment]
@mo-gr @vigneshshanmugam I would like to add tests but... where?

**Comment by vigneshshanmugam** [comment]
@rbarilani Guess that is fine since we need to be backwards compatible.

File: tests/fragment.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            async: attributes.async,
+            timeout: 4000,
+            custom: attributes.custom
+        };
+
+        const tag = { attributes };
+        const fragment = new Fragment(getOptions(tag));
+        const fattributes = fragment.attributes;
+
+        assert.equal(fattributes.id, expected.id);
```
Comment (vigneshshanmugam) [inline tests/fragment.js]: you can use assert.deepEquals to check these objects.

**Comment by rbarilani** [comment]
👍

**Comment by rbarilani** [comment]
@vigneshshanmugam updated and added a small README note regarding custom attributes.

**Comment by vigneshshanmugam** [comment]
👍

**Comment by phamvanthang310** [comment]
It seems custom attribute is no long pass to relevant function (`filterRequestHeaders`). 
https://github.com/zalando/tailor/blob/master/lib/fragment.js#L56

**Pull request merged**